modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ResizeHelper.java

```diff
@@ -56,7 +58,7 @@ public ResizeHelper(ResizeFeaturesBase rf,
         this.snap = (rf.getTableControl().isSnapToPixel() ? rf.getTableControl() : null);
```

Comment:
Obtaining the `Region` here is a bit hacky, this may be out of scope, but I would say `snap` should be a boolean, and the `snapXXX` helper methods in this class should call `ScaledMath`.

Reply:
Here i would disagree.  I specifically do not want to replicate the snap code in the Region, since the Region publishes its snapping API.  If there is a change in the snapping APIs in the Region, that change would need to be replicated again here, which I think is a bad idea.